eos-phone-home-if-up

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+[ "$IFACE" != "lo" ] || exit 0
```

Comment:
It's more for debugging than anything else. Having the script just magically exit (even if it's a success) makes bugs hard to locate/identify.